NOTICE-fips.txt

```diff
@@ -7049,285 +7049,447 @@ Contents of probable licence file $GOMODCACHE/github.com/prometheus/common@v0.62
 
 
 --------------------------------------------------------------------------------
-Dependency : github.com/prometheus/procfs
-Version: v0.15.1
-Licence type (autodetected): Apache-2.0
+Dependency : github.com/shirou/gopsutil/v4
```

Comment:
This change should not be in this PR.  The fix is in https://github.com/elastic/fleet-server/pull/5002.